python/pyspark/ml/stat.py

```diff
@@ -37,8 +37,8 @@ class ChiSquareTest(object):
 
     """
     @staticmethod
-    @since("2.2.0")
-    def test(dataset, featuresCol, labelCol):
+    @since("3.1.0")
```

Comment:
@huaxingao, let's don't change when the version was added. Also, `@since` adds `versionadded` automatically into the docstring so I think we can remove it in the docstring.

In this case, we could add `verstionchanged` directive instead to describe the difference happened in Spark 3.1.0.

Reply:
@HyukjinKwon Thanks for your comment. I changed back the version and also added  ```versionchanged``` directive. ```versionadded``` is for ```class ChiSquareTest```, not for method ```test```, so I guess I will leave it as is?